### DISCUSSION TITLE
Thank you for this great extension. [Suggesting improvements]

### DISCUSSION THREAD

**Discussion by kyroluskamal** [community/community#13831] [Copilot Conversations]
I used copilot recently. Actually, it is a mazing extension, so, I decided to help in improvements by writing my feedback.

I am using Angular in VScode
One thing I notice when writing import statement in angular is that copilot suggest some subfolders that are not exist. I think it will improvements if there is ability to read the subfolders in the node_module packages to suggest the right subfolder.

**Comment by pcgandhinj466** [comment]
On Tue, Mar 29, 2022 at 8:46 PM Kyrolus Kamal Faheem <
***@***.***> wrote:

> I used copilot recently. Actually, it is a mazing extension, so, I decided
> to help in improvements by writing my feedback.
>
> I am using Angular in VScode
> One thing I notice when writing import statement in angular is that
> copilot suggest some subfolders that are not exist. I think it will
> improvements if there is ability to read the subfolders in the node_module
> packages to suggest the right subfolder.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/feedback/discussions/13831>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARCYCKCU25QBA52JKO7MN5DVCOP4PANCNFSM5SAF4ZCA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Pravin Gandhi
